invenio_cli/cli/packages.py

```diff
@@ -67,11 +67,26 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages()
+
+    on_fail = "Some of the packages need to be updated."
+    on_success = "All packages are up to date."
+
+    run_steps(steps, on_fail, on_success)
 
 
 @packages.command()
+@click.argument("packages", nargs=-1, type=str)
 @pass_cli_config
-def update(cli_config):
-    """Update a single Python python package."""
-    raise click.UsageError("Not supported yet.")
+def update(cli_config, packages=[]):
```

Comment:
major: never use `{}` or `[]` as default in parameters... They are *mutable* and can lead to unexpected errors...

See this https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/